docs/UserGuide.md

```diff
@@ -120,127 +140,221 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
+| **`[mo/MONTH]`** | Allows you to customize the summary statistic to only consider entries in a specified month in `YYYY-mm` format, e.g. `2022-01` |
+
+#### Summary of all entries in the currently shown list
+
+- Example: `summary`
+- Expected:
+    <pre>
+       Total Expenditure: $154.49
+       Total Income: $250.00
+       Total Balance: $95.51
+    </pre>
+
+![SummaryCommand](images/ug/SummaryCommand.png)
+
+#### Summary of all entries the specified month
+
+- Examples: `summary mo/2022-10`
+- Expected:
+    <pre>
+          Financials Summarized
+          Total Expenditure: $4.20
+          Total Income: $250.00
+          Total Balance: $245.80
+    </pre>
+
+![SummaryCommandMonth](images/ug/SummaryCommandMonth.png)
+
+- Provides a financial summary recorded by the user in a month. The month refers to the month that is displayed to the user.
+- The `MONTH` field is optional, if no month is specified, the application displays the summary for all entries.
+
+### View entries by category: `view t/ENTRY_TYPE`
+
+| Input              | Explanation                                                              |
+|--------------------|--------------------------------------------------------------------------|
+| **`t/ENTRY_TYPE`** | An entry type refers to either `e` for "Expenditure" or `i` for "Income" |
+
+1. View a PieChart of all expenditures by categories 
+   - Examples: `view t/e`
+   - Expected: `Show graphically all expenditure by category` and a PieChart on the right of the application.
+   
+   ![ViewCommandExpenditureCategory](images/ug/ViewCommandExpenditureCategory.png)
+
+2. View a PieChart of all incomes by categories
+   - Examples: `view t/i`
+   - Expected: `Show graphically all income by category` and a PieChart on the right of the application
+
+### View entries by month: `view t/ENTRY_TYPE mo/MONTH`
+
+| Input              | Explanation                                                                              |
+|--------------------|------------------------------------------------------------------------------------------|
+| **`t/ENTRY_TYPE`** | An entry type refers to either `e` for "Expenditure" or `i` for "Income"                 |
+| **`mo/MONTH`**     | Allows you to view only entries in a specified month in `YYYY-mm` format, e.g. `2022-01` |
+
+1. View a LineGraph of all expenditures in a specified month 
+   - Examples: `view t/e mo/2022-01` where we specify the month to be January 2022.
+   - Expected: `Show graphically all expenditure by month` and a LineGraph on the right of the application
+
+   ![ViewCommandExpenditureMonth](images/ug/ViewCommandExpenditureMonth.png)
+   
+2. View a LineGraph of all incomes in a specified month
+   * Examples: `view t/i mo/2022-01` <br>
    Expected: `Show graphically all income by month` and a LineGraph on the right of the application
```

Comment:
small nit to pick, since everything is changed to a '-', we should follow the format for these 2 lines as well